include/RAJA/Timer.hxx

```diff
@@ -91,7 +91,7 @@ public:
                  + std::chrono::microseconds(tstart.tv_usec);
     auto stop = std::chrono::seconds(tstop.tv_sec)
                 + std::chrono::microseconds(tstop.tv_usec);
-    telapsed += std::chrono::duration<double>(stop - start).count();
+    telapsed += std::chrono::duration<long double>(stop - start).count();
```

Comment:
Nifty, I don't think this is worth making a huge thing over, just having a note/issue sounds like the right reaction.